docs/03-tutorials/00-api-exposure/apix-04-expose-and-secure-workload-jwt-istio-ext-auth.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Expose and secure a workload with Istio external auth
+---
+
+This tutorial shows how to expose workload using Istio authorization policy and secure it using Istio RequestAuthentication with JWT token
+
+## Prerequisites
+
+To follow this tutorial, use Kyma 2.0 or higher.
+
+This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create one of those, follow the [Create a workload](./apix-02-create-workload.md) tutorial.
+
+## Expose your workload using VirtualService
+
+1. Export environment variable with your namespace:
+
+   ```shell
+   export NAMESPACE={NAMESPACE} # e.g. default
+   export DOMAIN={DOMAIN_NAME} # This is a Kyma domain or your custom subdomain e.g. api.mydomain.com.
+   ```
+
+1. Run:
+
+   ```shell
+   cat <<EOF | kubectl apply -f -
+   apiVersion: networking.istio.io/v1alpha3
+   kind: Gateway
```

Comment:
gateway creation is handled in https://kyma-project.io/docs/kyma/main/03-tutorials/00-api-exposure/apix-02-create-workload/